src/containers/User/Login.tsx

```diff
@@ -15,10 +15,16 @@ const Login: React.FC = () => {
   const [isError, setIsError] = useState(false)
   const [networkError, setNetworkError] = useState('')
   const [loginPayload, setLoginPayload] = useState<LoginPayload>()
-  const [selectedOrgIndex, setSelectedOrgIndex] = useState(0)
+  const [selectedOrgId, setSelectedOrgId] = useState<number>(-1)
   const { push, history } = useRouter()
   const { onLogin } = useUserState()
 
+  const noOrgOption: OrganisationSimple = {
+    orgId: 0,
```

Comment:
can 0 and -1 be constants (LOGIN_AS_NO_ORG, NO_ORG_SELECTED)

Reply:
a36683d8